src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -3674,12 +3674,15 @@ def build_bicep_file(cmd, file, stdout=None, outdir=None, outfile=None, no_resto
         print(output)
 
 
-def publish_bicep_file(cmd, file, target):
+def publish_bicep_file(cmd, file, target, documentationUri=None):
     ensure_bicep_installation()
 
     minimum_supported_version = "0.4.1008"
     if bicep_version_greater_than_or_equal_to(minimum_supported_version):
-        run_bicep_command(cmd.cli_ctx, ["publish", file, "--target", target])
+        args = ["publish", file, "--target", target]
+        if documentationUri:
+            args += ["--documentationUri", documentationUri]
```

Comment:
Can we add another version check for the parameter?

Reply:
Sure, updated here:
https://github.com/Azure/azure-cli/pull/25461/commits/3625f03e24a57d5c2e4700c43262de247f6126d0

Thanks!

Reply:
Should we reject the parameter and log an error instead of ignoring it if the version is lower than 0.14.46?

Reply:
Actually that makes sense. Updated here:
https://github.com/Azure/azure-cli/commit/732be83e138e351745c9d2ab62bad655dd069541

![image](https://user-images.githubusercontent.com/30270536/219153294-d36146a8-cbf1-4e79-b6e4-7af3d67a4607.png)